reference/hoon-expressions/rune/bar.md

```diff
@@ -8,12 +8,77 @@ Core expressions produce cores. A core is a cell of `[battery payload]`.
 The `battery` is code, a battery of Nock formulas.  The `payload` is the data
 needed to run those formulas correctly.
 
-Four core runes (`|=`, `|.`, `|-`, and `|*`) produce a
+Five core runes (`|=`, `|.`, `|-`, `|*`, and `|$`) produce a
 core with a single arm, named `$`. We can recompute this arm
-with changes, which useful for recursion among other things:
+with changes, which is useful for recursion among other things.
 
 ## Runes
 
+### |$ "barbuc"
+
+Produces a wet gate that creates a structure parameterized by molds.
```

Comment:
>structure

Might be better to more explicitly call the result a type/mold? Same applies below.

Reply:
`|$` produces a mold builder, which is a wet gate with one or more molds as its sample, producing a mold.

Reply:
Thanks, I had a feeling that I wasn't saying what it produced quite right. I've changed the wording everywhere to say it produces a mold builder.

https://github.com/urbit/docs/pull/877/commits/1d189072daccbf7ade5851682608a4e11163ba25